### ISSUE TITLE
Mobile Issue - Writer text area scrolling out of view with Keyboard Focus

### ISSUE THREAD

**Issue by joetex** [forem/forem#4385]
**Is your feature request related to a problem? Please describe.**
Android Firefox text area will scroll out of view when moving text cursor or when the text area gains focus.

**Describe the solution you'd like**
If text cursor is in view, do not scroll.  
If text cursor goes out of view, slide it back into view.

**Describe alternatives you've considered**
Android Chrome doesn't scroll at all, so if the cursor is towards bottom of screen, the keyboard and UI will block the text area.

**Additional context**

Android Firefox issue as gif:
https://gfycat.com/bravedelightfulabyssiniangroundhornbill

Android Chrome issue as gif:
https://gfycat.com/vengefulglisteningcheetah

**Comment by tudorpavel** [comment]
I want to help with this but of course there isn't an easy fix.

There are issues on Android browsers when using dynamic element heights and/or fixed positioning while opening the keyboard. The reason is that when opening the keyboard on Android browsers, the body height will change by subtracting the keyboard height. Dynamic heights will change while the keyboard is opening and cause the browser's "autoscroll to cursor" behavior to go nuts.

I managed to make the behavior a bit better by changing the layout:
  - I removed `preact-textarea-autosize`, added a hardcoded `height: 300px` to the textarea and re-introduced the scroll
  - bottom buttons are no longer with `position: fixed`

I'm not sure what else needs to change, for example I noticed that sometimes there's a `v1` for `articleform__form articleform__form--v2` about which I'm not sure but we can discuss that in a WIP PR.

Result in Firefox:
![firefox](https://user-images.githubusercontent.com/2506871/67224577-30789480-f43a-11e9-8f90-862f9f319f0a.gif)


P. S.

> If text cursor goes out of view, slide it back into view.

I've been an Android user for many years and this wpuld not be my expectation. The behavior of the cursor while scrolling a textarea is also browser-dependent and not something we can change as far as I know.
